src/main/java/kostka/moviecatalog/service/DbMovieService.java

```diff
@@ -63,15 +64,22 @@ public Movie getMovie(final Long movieId) {
         return movieRepository.findByIdInOrderByIdDesc(ids);
     }
 
-    public List<Movie> get5LatestMovies() {
+    public List<Movie> get5LatestMoviesFromCache() {
         List<Long> longIds = getMovieIdsFromRedisCache(LATEST_MOVIES_KEY);
         return movieRepository.findByIdInOrderByIdDesc(longIds);
     }
 
     public List<Movie> getTop5RatingMoviesFromDB() {
-        return movieRepository.findTop5ByOrderByRatingDesc();
+        List<Movie> movies = movieRepository.findTop5ByOrderByRatingDesc();
+        return movies.isEmpty() ? new ArrayList<>() : movies;
```

Comment:
K čemu přesně je tady ten ternární operátor?
Když je movies prázdné, tak stejně vrátíš prázdný seznam.

Nestačí dát jen `return movies`?

Reply:
jasne, zapomnel jsem to z tama vyhodit, diky -> predtim jsem tam mel nejakou vyjimku, ze "dbMovieService.get5LatestMoviesFromDB()" nesmi byt null nebe tak neco, kterou jsem chtel timto vyresit, ale stejne nevyresil, takze ji odchytavam v try catch bloku:

```
try {
            redisService.updateLatestMovies(dbMovieService.get5LatestMoviesFromDB());
        } catch (Exception e) {
            LOGGER.info("[Scheduled task] No movie in DB");
        }
```

Reply:
takze mas pravdu, staci :)